assets/src/dashboard/theme.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createGlobalStyle, ThemeContext } from 'styled-components';
+import { useContext } from 'react';
+
+/**
+ * Internal dependencies
+ */
+export const GlobalStyle = createGlobalStyle`
+  @import url(//fonts.googleapis.com/css?family=Google+Sans);
+	*,
+	*::after,
+	*::before {
+		box-sizing: border-box;
+    }
+    
+    h1 {
+      font-family: 'Google Sans', 'Roboto', sans-serif;
```

Comment:
Don't you also need to load Roboto?

Reply:
Roboto is getting added through the font loader in php as far as i can tell. Since Carlos needs this underlying structure I'll add that separately if you're ok with it.

Reply:
I also do not see any usages of Roboto throughout the design of the Dashboard UI. But Roboto is usable before this PR.

Reply:
sounds good, merging this one for now.